.gitignore

```diff
@@ -7,7 +7,8 @@ generated/
 /src/test/javascript/coverage/
 /src/test/javascript/PhantomJS*/
 keystore.jks
-keystore.p12
```

Comment:
just `keystore.p12` is intentional. Any file that should then not be ignored is intentional.

Reply:
Oops... Reverted my unintentional change.